packages/input-date/test/lion-input-date.test.js

```diff
@@ -141,9 +141,9 @@ describe('<lion-input-date>', () => {
   it('serializes to iso format', async () => {
     const el = await fixture(
       html`
-        <lion-input-date .modelValue="${new Date('2000-12-12')}"></lion-input-date>
+        <lion-input-date .modelValue="${new Date('2000-12-15T00:00:00')}"></lion-input-date>
```

Comment:
```suggestion
        <lion-input-date .modelValue="${new Date('2000/12/15')}"></lion-input-date>
```

I think that is better-supported cross-browser... can we test if that works as well?

Reply:
Also works, updated in new commit.